eng/pipelines/notifications.yml

```diff
@@ -11,6 +11,7 @@ stages:
 
   jobs:
   - job: Run
+    timeoutInMinutes: 120
```

Comment:
Why did we need to extend the timeout?

Reply:
Based on recent history, we use more time on AzureDevOpsService call, which made some of the run timeout.

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1355827&view=results